courses/free-courses-en.md

```diff
@@ -805,6 +805,7 @@
 ### Misc
 
 * [Advanced Adobe XD (Web Design)](https://webdesign.tutsplus.com/courses/advanced-adobe-xd-for-everyone) - Adi Purdila
+* [Art of Code](https://www.youtube.com/watch?v=6avJHaC3C2U&list=PUTdw38Cw6jcm0atBPA39a0Q) - Dylan Beattie (Youtube)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Hi @eshellman. The youtube link is a playlist link? And the name of the collaborator / creator is behind the link. Dylan Beattie. 
Do you have an example of how you exactly want it? :) 

Thanks!

Reply:
Just 1h video. enough for me

Fixing YT url, title and notes...

```suggestion
* [The Art of Code - Dylan Beattie](https://www.youtube.com/watch?v=6avJHaC3C2U) - Dylan Beattie (YouTube)
```

Reply:
... and you can omit the (YouTube)

Reply:
> ... and you can omit the (YouTube)

Already done by @Fazer01